deployment/factorycube-server/values.yaml

```diff
@@ -51,7 +51,7 @@ mqtttoblob:
   enabled: true
   image:
     repository: unitedmanufacturinghub/mqtt-to-blob
-    tag: testing
```

Comment:
Let's just remove the tags entirely as then we have the default tag handling kicking in 

See also:  https://github.com/united-manufacturing-hub/united-manufacturing-hub/blob/5177804744f0ff0294b822e85b4967655c2bc9fb/deployment/factorycube-server/templates/mqtttoblob-deployment.yaml#L36-L40 

The default behavior means that it will use the latest release tag (currently v0.5.0). With this we prevent the issue with latest, which we had e.g. with VerneMQ.